tests/test_executors.py

```diff
@@ -5,9 +5,27 @@
 
 # %%
 from protein_search.backend.my_executors import *
+from jina import DocumentArray, Flow
+from jina.types.document.generators import from_csv
 
 # %%
 def test_ProtBERT_encoding():
     sequences = ["A E T C Z A O", "T E A B Z A O"]
     docs = DocumentArray((Document(content=seq) for seq in sequences))
     embedded_docs = ProtBertExecutor().encode(docs)
+
+# %%
+def test_Indexer_indexing_samelength():
+    proteins = DocumentArray(
+        from_csv(
+            fp=open("../data/samelength.csv"),
```

Comment:
Tests are run from the parent directory, so this should be `fp=open("data/samelength.csv")`. This should fix the `build` badge (as tests in the cloud are failing for this reason, I believe).